### PR TITLE
feat(birdie): favorite files feature

### PR DESCRIPTION
**Pull request by sylviacx**
#6770525631

### CODE REVIEW

**Comment by sylviacx** [comment]
TODO: reconcile config and files already downloaded, ability to unfavorite files

**Comment by sylviacx** [comment]
TODO: reconcile paths -> directories when all files within a directory are added to lfs.fetchinclude

File: Cargo.toml
Comment (BelieverCoho) [inline Cargo.toml:1]: intended removal of these 2? :

Comment (sylviacx) [reply Cargo.toml:1]: not intended you're right, just fixed!

**Review by rudoi — APPROVED** [review]
let's go!!!!

**Pull request merged**